doc/classes/HeightMapShape3D.xml

```diff
@@ -4,11 +4,13 @@
 		A 3D height map shape used for physics collision.
 	</brief_description>
 	<description>
-		A 3D heightmap shape, intended for use in physics. Usually used to provide a shape for a [CollisionShape3D]. This is useful for terrain, but it is limited as overhangs (such as caves) cannot be stored. Holes in a [HeightMapShape3D] are created by assigning very low values to points in the desired area.
+		A 3D heightmap shape, intended for use in physics. Usually used to provide a shape for a [CollisionShape3D]. This is useful for terrain, but it is limited as overhangs (such as caves) cannot be stored. Holes in a [HeightMapShape3D] are created by assigning the value [constant @GDScript.NAN] to points in the desired area. This allows placing objects in the holes that can be used to model overhangs, caves, interiors and so on.
 		[b]Performance:[/b] [HeightMapShape3D] is faster to check collisions against than [ConcavePolygonShape3D], but it is significantly slower than primitive shapes like [BoxShape3D].
-		A heightmap collision shape can also be build by using an [Image] reference:
+		A heightmap collision shape can also be built by using an [Image] reference:
 		[codeblocks]
 		[gdscript]
+		# EXR or HDR formats are recommended for heightmap images, as they support 32 bpc floating-point values.
+		# Formats that can only be imported as 8 bpc by Godot are not recommended due to their limited precision.
```

Comment:
Using a heightmap with 16-bit or 32-bit data, stored in EXR or HDR format is recommended. Using 8-bit height data, or a format like PNG that Godot imports in 8-bit, will result in a terraced terrain.